build.xml

```diff
@@ -1035,31 +1036,44 @@ ext = BeanUtils.cook(ext);
                             <substitution expression=" "/>
                             <fileset file="@{var}${nodebug.ext}"/>
                         </replaceregexp>
-                        <echo>Minimizing to @{var}${jsmin.ext}</echo>
-	                    <try>
-	                        <block>
-                                <java jar="${build.tmp.dir}/yuicompressor-2.4.8.jar" fork="true" failonerror="true"
-		                              output="${build.dir}/yui-compress.out">
-                                    <!-- Add error="@{var}${compresserror.ext}" in java task if *.err files need to be generated. -->
-		                            <arg value="--line-break"/>
-		                            <arg value="0"/>
-		                            <arg value="--type"/>
-		                            <arg value="js"/>
-		                            <arg value="-o"/>
-		                            <arg value="@{var}${jsmin.ext}"/>
-		                            <arg value="@{var}${nodebug.ext}"/>
-                                    <arg value="--verbose"/>
-		                        </java>
-	                        </block>
-		                    <catch>
-			                    <antcall target='compress-test-files'>
-				                    <param name='filename' value='@{var}${depends.ext}' />
-			                    </antcall>
-		                    </catch>
-		                    <finally>
-			                    <!-- do nothing -->
-		                    </finally>
-	                    </try>
+                        <if>
+                            <or>
+                                <equals arg1="@{var}" arg2="${build.js.dir}/JQuery_all.js" />
+                                <equals arg1="@{var}" arg2="${build.js.dir}/Clipboard_all.js" />
+                            </or>
+                            
+                            <then>
+                                <echo>Copying thirdparty library file to @{var}${jsmin.ext}</echo>
+                                <copy tofile="@{var}${jsmin.ext}" file="@{var}"/>
+                            </then>
+                            <else>
+                                <try>
+                                    <block>
+                                        <echo>Minimizing to @{var}${jsmin.ext}</echo>
+                                        <java jar="${build.tmp.dir}/yuicompressor-2.4.8.jar" fork="true" failonerror="true"
+                                            output="@{var}${compressoutput.ext}">
+                                            <!-- Add error="@{var}${compresserror.ext}" in java task if *.err files need to be generated. -->
```

Comment:
Do we need this commented code?

Reply:
For future reference the message was kept , so that if we want to generate error files ,lets keep it.